docs/slack.rst

```diff
@@ -174,54 +174,130 @@ Our goal with Write the Docs is to be **the most welcoming community on the inte
+  Be up front about paywalls, registration requirements, referral benefits, or commissions.
+  If you're running a survey, say when and how the results will be published.
+- **Give more than you take.**
+  Make self-promotion a reasonable proportion of your participation in the community.
+  Too much self-promotion creates distrust and hostility.
+  If you're not sure if it's too much, aim to help others 10 times for every time you self-promote.
+- **Post publicly instead of sending direct messages (DMs)**.
+  A DM is like a home and you're like a vampire: you're not allowed in without an invitation.
+  It can be tempting to DM people, but please respect their inboxes and post in public channels unless invited to DM.
+- **Read the room.**
+  The Write the Docs Slack is informal and conversational; high-pressure sales aren't welcome.
+  If people aren't interested in what you're offering, don't take it as a license to become louder and more aggressive.
+  If you're asked to stop, then stop.
+
+Here are some channels specifically for wholesome self-promotion:
+
+- ``#job-posts-only`` and ``#freelance`` are the best places to paid jobs
+- ``#community-help-wanted`` is a good place to share calls for participation on open-source projects
+- ``#???`` is a good place to announce non-Write the Docs events
+- ``#???`` is a good place to share a survey or calls for research interview subjects
+- ``#community-showcase`` is a space for shame-free self-promotion
 
-- **No Direct Pitches** - Our Slack is not the place for sales pitches. Consider sponsoring one of our :doc:`conferences </conf/index>` instead!
-- **No links without context** - If you put a link in the chat without context or explanation, it may be deleted.
-- **No Brands** - If you're going to post, post as yourself, not as a brand. We are a community of individuals and we all exist outside of our day jobs and personal brands.
-- **Be Transparent** - Tell people clearly what you are promoting, why it matters to documentarians, and whether you are personally or professionally involved. Be up front if there are any paywalls or registration gates in front of the content. Avoid using link shorteners. These efforts help distinguish spam or drive-by marketing from actual community engagement.
+Reporting Issues
+~~~~~~~~~~~~~~~~
 
 If you see a message in a public channel that looks like spam, you can react to it with the ``:spam:`` reactji to bring the moderators' attention to it.
 
-Avoid Microagressions
-~~~~~~~~~~~~~~~~~~~~~
+If you see people not following the behavior guidelines you can:
 
-We recommend reading this `Microaggressions:  Making space for everybody <https://www.metafilter.com/microaggressions.mefi>`_ article that goes over how to avoid microaggressions when dealing with people in our community.
+* Message them privately, and explain why what they said might have been hurtful to someone, referring to this guide.
```

Comment:
This is an exception to "don't DM without invite"? hmmm

Reply:
Yeah, I was thinking about that too. Revised this a bit with 0e77b401.

Reply:
It's tricky, because some folks might well respond better to a private "have you thought about..." instead of public "chastisment"...

Reply:
I agree with that. I think a few things come together here:

- We only explicitly advise against DMs for self-promotion. I think folks engaged in other kinds of conversations can and will use their judgement about taking it to DMs.
- For this particular case, I dropped the suggestion to DM. But I don't think we're forbidding it either.
- Sometimes a public chastisement is in the community's interest. Stating our values in public is a way to communicate to the victims of bad actors as well the bad actors themselves. But I'm skeptical we can train the community, in general, to handle those nuances in a document like this.